src/content/self-hosted/gke.mdx

```diff
@@ -119,6 +119,7 @@ Download a copy of the [Okteto GKE configuration file](/docs/self-hosted/gke/con
 - Your `license` (optional)
 - A `subdomain`
 - `clientId` and `clientSecret` of the OAuth 2.0 Client you created
+- Your cluster's API server endpoint
```

Comment:
Can we link to a google doc on how to get it?

Reply:
I cannot find anything. I am going to merge because right now the guide is broken. But I want to add the link later, also for the rest of providers